src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -366,6 +366,12 @@ private static void validateCheckSection(ModuleFactory moduleFactory, String fil
             final String subSectionName = subSection.getAttributes().getNamedItem("name")
                     .getNodeValue();
 
+            // following subsections of Checker are not typical
+            if ("Checker".equals(sectionName)
+                    && ("Example of Usage".equals(subSectionName)
```

Comment:
Why do we need `Example of Usage` override?
URLs generated by junit are good and point right to the line.